legendary/cli.py

```diff
@@ -580,7 +580,8 @@ def launch_game(self, args, extra):
             return self._launch_origin(args)
 
         igame = self.core.get_installed_game(app_name)
-        if (not igame or not igame.executable) and (game := self.core.get_game(app_name)) is not None:
+        game = self.core.get_game(app_name)
+        if (not igame or not igame.executable) and game is not None:
```

Comment:
Why was this changed?

Reply:
so that game is accessible by get_cdn_urls call below

Reply:
Ah right, forgot that the operator worked like that. ...I should really write more Python again